ChangeLog

```diff
@@ -1,3 +1,5 @@
+	* use DSCP_TRAFFIC_TYPE socket option on windows
+	* update default ToS setting according to RFC 8622
```

Comment:
I don't see this change in RC_2_0 or in this PR

Reply:
it's from a PR landed earlier today. it hasn't been merged into RC_2_0 yet.